dcpandas/df_builder.py

```diff
@@ -39,35 +39,28 @@ def build_time_series(place, stat_var):
     return pd.Series(dc.get_stat_series(place, stat_var))
 
 
-def _group_stat_all_by_obs_options(places, stat_vars, mode):
-    """Groups the result of `get_stat_all` by Observation options for time series.
+def _group_stat_all_by_obs_options(places, stat_vars, time_series=True):
+    """Groups the result of `get_stat_all` by Observation options for time series or covariates.
     
     Args:
       places (`str` or `iterable` of `str`): The dcids of Places to query for.
       stat_vars (`Iterable` of `str`): The dcids of the StatisticalVariables.
-      mode (`str`): "series" to output time series grouped by Observation options, or
-        "covariates" to output latest Observations.
+      mode (`boolean`): if True, output time series grouped by Observation options; if False, output latest Observation grouped by Observation options.
```

Comment:
In this case, consider naming it is_series or keep_series. It'll make the rest of the code more readable.

Reply:
Thanks for the catch and suggestion! Changed.